Mathlib/MeasureTheory/Function/LpSpace.lean

```diff
@@ -788,6 +806,18 @@ theorem norm_indicatorConstLp_top (hμs_ne_zero : μ s ≠ 0) :
     ENNReal.rpow_zero, mul_one, ENNReal.coe_toReal, coe_nnnorm]
 #align measure_theory.norm_indicator_const_Lp_top MeasureTheory.norm_indicatorConstLp_top
 
+open scoped symmDiff in
+/-- Compute the `ℒᵖ` norm of the difference of two constant indicators of sets with finite measure
+and same constant using symmetric difference. -/
+theorem norm_indicatorConstLp_sub {t : Set α} (ht : MeasurableSet t) (hμt : μ t ≠ ∞)
```

Comment:
```suggestion
theorem norm_indicatorConstLp_sub {t : Set α} {ht : MeasurableSet t} {hμt : μ t ≠ ∞}
```
Since the situation is symmetric between `s` and `t` the explicitness of the assumptions should match. The reason everything can be implicit here is that most of the time you will rewrite from left to right, hence the hypotheses needed already apply in the expression.

Reply:
I wrote it this way because it was done that way in similar theorems below (e.g. [MeasureTheory.dist_indicatorConstLp_eq_norm](https://leanprover-community.github.io/mathlib4_docs/Mathlib/MeasureTheory/Function/LpSpace.html#MeasureTheory.dist_indicatorConstLp_eq_norm)), should I change those too? And in fact I realized this result gave me a much simpler proof, but makes me question the interest of the theorem.

Reply:
Indeed I think these should be changed as well (and that should be easy since they're not used). Regarding the usefulness of the theorem, indeed I think the existing result is somewhat better as it's the best statement you can hope for without side conditions, and doing two rewrites instead of one is not a big deal in practice. Would you be okay with dropping this in favor of the existing result?

Reply:
Yes I'll do that!

Reply:
However I did not find a lemma to rewrite the norm as a distance, because `Lp E p mu` is not a normed group if `p < 1` so [dist_eq_norm](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Analysis/Normed/Group/Basic.html#dist_eq_norm) does not work, do you see a better way than doing a `change`?

Reply:
I just added this as a theorem.

Reply:
Yes I think that was the best thing to do, we already have some duplicates like [MeasureTheory.Lp.norm_neg](https://leanprover-community.github.io/mathlib4_docs/Mathlib/MeasureTheory/Function/LpSpace.html#MeasureTheory.Lp.norm_neg).